modules/development/pages/kubernetes-vm.adoc

```diff
@@ -0,0 +1,68 @@
+= Kubernetes VM
+:toc:
+:toclevels: 3
+
+WARNING: This page is a work in progress.
+
+. Copy the kubernetes VM image from `node-images` into the management VM.
++
+[source,bash]
+----
+scp node-images/output-kubernetes-vm-qemu/kubernetes-vm-x86_64.qcow2 root@192.168.0.2:~/
+----
+. Login to the Management VM
+
+There are two options to get started. If you're still on the 1st Hypervisor, you can login using the isolated network. The address of the management-vm will be 192.168.0.2 or whatever value was assigned in the management-vm configuration steps.
+
+[source,bash]
+----
+ssh root@192.168.0.2
+----
+. Move the kubernetes VM into the proper location
++
+[source,bash]
+----
+mkdir -p /vms/images
+mv ~/kubernetes-vm-x86_64.qcow2 /vms/images/
+----
+. Generate an SSH key
++
+[source,bash]
+----
+ssh-keygen
+----
+. Copy SSH key to the host system
++
+[source,bash]
+----
+ssh-copy-id 192.168.0.2
+----
+. Change to the working directory for deploying the first Kubernetes Master VM
++
+[source,bash]
+----
+cd /srv/cray/terraform/kubernetes-master/
+----
+. Set the var for the QEMU system URI
++
+[source,bash]
+----
+qemu+ssh://192.168.0.2/system
+----
+. Review the plan for terraform
++
+[source,bash]
+----
+terraform plan
+----
+. Execute terraform
++
+[source,bash]
+----
+terraform apply
+----
+
+
+
+
+
```

Comment:
Clean up the extra new-lines, double-check other files too... I saw some extra new-lines here and there but for the most part this PR was tidy.